index.html

```diff
@@ -103,10 +103,7 @@
 
 <body>
   <div class="container">
-  <h2>Registration for the 2019 Winter Berkeley DESC meeting</h2>
-  The registration fee for the collaboration meeting is <u><i>$100</i></u>, which
-  covers attendance to the 5-day meeting and coffee breaks. Fees do not cover a
-  Collaboration Meeting Dinner, which is still undetermined.
+  <h2>Registration for the 2020 Summer DESC meeting</h2>
```

Comment:
```suggestion
  <h2>Registration for the 2020 July DESC meeting</h2>
```
We have an opportunity to avoid northern-hemisphere-centric language (with thanks to a reminder from Andres!)

Reply:
>.< good catch, sorry I didn't think of that... it's hard to grasp that there is actually an entire world outside of my room.